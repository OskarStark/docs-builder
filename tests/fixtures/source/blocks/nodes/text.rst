**Lorem ipsum** dolor sit amet, consectetur adipisicing elit. Ut enim ad minim
veniam, quis nostrud *exercitation ullamco laboris* nisi ut aliquip ex ea
commodo consequat. ``Duis aute irure dolor`` in reprehenderit in voluptate
velit esse cillum dolore eu fugiat nulla pariatur.

Excepteur ~1,000 sint! Occaecat cupidatat non proident, sunt in culpa qui officia
deserunt mollit anim id est laborum. áàâäãåéèêëíìîïóòôöõúùûüñçÿ
