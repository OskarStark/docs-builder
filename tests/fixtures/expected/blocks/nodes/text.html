<p><strong>Lorem ipsum</strong> dolor sit amet, consectetur adipisicing elit. Ut enim ad minim
veniam, quis nostrud <em>exercitation ullamco laboris</em> nisi ut aliquip ex ea
commodo consequat. <code translate="no" class="notranslate">Duis aute irure dolor</code> in reprehenderit in voluptate
velit esse cillum dolore eu fugiat nulla pariatur.</p>

<p>Excepteur ~1,000 sint! Occaecat cupidatat non proident, sunt in culpa qui officia
deserunt mollit anim id est laborum. áàâäãåéèêëíìîïóòôöõúùûüñçÿ</p>
